src/lib/ecl/attitude_fw/ecl_yaw_controller.cpp

```diff
@@ -158,71 +176,80 @@ float ECL_YawController::control_bodyrate_impl(const struct ECL_ControlData &ctl
 	/* get the usual dt estimate */
 	uint64_t dt_micros = ecl_elapsed_time(&_last_run);
 	_last_run = ecl_absolute_time();
-	float dt = (float)dt_micros * 1e-6f;
+	float dt = (float) dt_micros * 1e-6f;
 
 	/* lock integral for long intervals */
 	bool lock_integrator = ctl_data.lock_integrator;
-	if (dt_micros > 500000)
+
+	if (dt_micros > 500000) {
 		lock_integrator = true;
+	}
 
 	/* input conditioning */
 	float airspeed = ctl_data.airspeed;
+
 	if (!isfinite(airspeed)) {
-	/* airspeed is NaN, +- INF or not available, pick center of band */
-	airspeed = 0.5f * (ctl_data.airspeed_min + ctl_data.airspeed_max);
+		/* airspeed is NaN, +- INF or not available, pick center of band */
+		airspeed = 0.5f * (ctl_data.airspeed_min + ctl_data.airspeed_max);
+
 	} else if (airspeed < ctl_data.airspeed_min) {
-	airspeed = ctl_data.airspeed_min;
+		airspeed = ctl_data.airspeed_min;
 	}
 
-
```

Comment:
why did you remove the the influence of the other axes in the body rate setpoint? Why not subtract the calculated value from the bodyrate setpoint?